tasks/download.sh

```diff
@@ -11,3 +11,35 @@ else
   printf '%s\n' "Downloading: ${PT_source}" >&2
   curl -f -L -o "$PT_path" "$PT_source"
 fi
+
+if [[ "$PT_check_download" != "true" ]]; then
```

Comment:
Usually, the negative conditions are hard to read, and easy to misread. 

Should we change that to something like:

```suggestion
if [[ "$PT_check_download" == "false" ]]; then
```

That could work if we are 100% positive that we will have "false" as String, but if we could have Null or Empty, then things get tricky. Let me know what you think @timidri